### ISSUE TITLE
[CRM Module] Memory leak when work with customer that have custom field of Link type

### ISSUE THREAD

**Issue by andy-nix** [freescout-helpdesk/freescout#2447]
CRM module version: **1.0.17**

Hi,
I trying to use introduced in this version of module feature of custom field with type Link
After I added custom field (e.g. `https://my.site.com/{%cf_4%}`) and trying to open any customer - request to https://<freescout domain>/customers/2/edit?x_embed=1 freeze and return partial content after some time. 

After that in logs I see:
```
Maximum execution time of 30 seconds exceeded {"userId":1,"email":"my@email.com","exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 1): Maximum execution time of 30 seconds exceeded at /var/www/freescout/vendor/laravel/framework/src/Illuminate/Database/Connection.php:634)
```

Also during this request I see gradual reduction of memory on the server till execution time timeout are not reached

I was able identify that freeze happen in **./Modules/Crm/Entities/CustomerField.php** method `getCustomerFieldsWithValues()`, but I'm not sure how to fix this issue

Also I have another customer custom field of type Link that use placeholder `{%cf_8%}` and it somehow doesn't affect work with Customers.

Please help identify and fix this issue

**Comment by freescout-helpdesk** [comment]
Could you please send a screenshot of the link field causing the issue.

**Comment by andy-nix** [comment]
![Screenshot 2022-12-07 18-53-51](https://user-images.githubusercontent.com/43207471/206242354-dc9387cf-f2c3-4cd6-803a-71033529e272.png)

And the field from which it depends
![Знімок екрана з 2022-12-07 18-56-59](https://user-images.githubusercontent.com/43207471/206242581-679fc7f1-aecd-48d1-b67c-2e417869ae0d.png)

**Comment by freescout-helpdesk** [comment]
> Also I have another customer custom field of type Link that use placeholder {%cf_8%} and it somehow doesn't affect work with Customers.

Please send a screenshot of this field.

**Comment by freescout-helpdesk** [comment]
We've managed to reproduce it only when one link field references another link field. Fixed in CRM Module v1.0.18

**Comment by andy-nix** [comment]
Confirm, with CRM Module v1.0.18 everything works fine. Thank you
